regression-test/suites/compaction/test_compaction_dup_keys.groovy

```diff
@@ -0,0 +1,185 @@
+import org.codehaus.groovy.runtime.IOGroovyMethods
```

Comment:
why not move the "import org.xxx.yyy" statement under the license comments?

Reply:
Hi @carlvinhust2012 , sorry for late reply. This is generated by IDEA and in a wrong order. I modified this in PR #9672